apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,58 @@
+/* global DetailsUI, Details, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
+    ], function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+window.onload = function() {
+  var dependencies = [
+    '/contacts/style/webrtc-client/webrtc_client.css',
+    '/contacts/js/webrtc-client/webrtc_client.js',
+    '/contacts/services/contacts.js',
+    '/contacts/js/activities.js',
+    '/shared/js/contacts/utilities/event_listeners.js',
+    '/shared/js/l10n_date.js',
+    '/shared/js/contacts/import/utilities/config.js',
+    '/contacts/js/utilities/extract_params.js',
+    '/contacts/js/utilities/cookie.js',
+    '/shared/js/contact_photo_helper.js',
+    '/shared/js/contacts/contacts_buttons.js',
+    '/shared/js/text_normalizer.js',
+    '/shared/js/contacts/utilities/dom.js',
+    '/shared/js/contacts/utilities/templates.js',
+    '/shared/js/contacts/import/utilities/misc.js',
+    '/dialer/js/telephony_helper.js',
+    '/shared/js/contacts/sms_integration.js',
+    '/contacts/js/match_service.js',
+    '/contacts/views/details/js/details_ui.js',
+    '/contacts/views/details/js/details_controller.js',
+    '/contacts/js/fb_loader.js'
+  ];
+
+  LazyLoader.load(dependencies, function() {
+    fbLoader.load();
+    DetailsUI.init();
+    Details.init();
+    navigator.mozSetMessageHandler('activity', activity => {
+      Details.setActivity(activity);
+      var id = activity.source.data.params.id;
+
```

Comment:
Add a TODO here pointing to de 'Advanced' #open activity, due to this code can not handle VCards yet.